<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditObject</class>
 <widget class="QDialog" name="EditObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>531</width>
     <height>601</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#background {
background-color: rgb(170, 170, 150)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="namelabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Название:</string>
    </property>
   </widget>
   <widget class="QLabel" name="adresslabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
   <widget class="QLabel" name="square_podval_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Площадь подвала, м2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="koefpodvallabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Коэффициент подвала:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="adress">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="podva_square">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="koef_podvl">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="squarelabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Площадь, м2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="square">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>240</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nazavanie">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Редактирование Объекта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okey">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="podvalradio">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Подвал</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="koef_tech">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="koeftechlabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Коэффициент технического обустройства помещения:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
